locales-pending/dashboard-premium.ftl

```diff
@@ -0,0 +1,160 @@
+premium-badge-label = { -brand-premium }
+premium-cta-label = Upgrade to { -brand-premium }
```

Comment:
Q: Per @rhelmer's recent #3934 PR, should these be `-premium` or `-plus`?

Reply:
I believe that Flod suggested `brand-monitor-plus`, we were holding back landing that PR until we were closer to shipping but I believe we decided to just land it which should make this easier.

Reply:
@rhelmer 
Not sure I understand. 

> we decided to just land it which should make this easier.

"it" is which PR?

Reply:
> @rhelmer Not sure I understand.
> 
> > we decided to just land it which should make this easier.
> 
> "it" is which PR?

Sorry, I mean the re-branding PR https://github.com/mozilla/blurts-server/pull/3934 that introduces `brand-monitor-plus`.

Reply:
#3934 just landed and this PR might need a rebasin' since it looks like we have a few file conflicts now.

Reply:
Yep, I'll do that after approval so people don't lose track of their existing reviews.

Reply:
Alright, all conflicts are resolved, tests are fixed, approvals are in, so with a final rebase I think this is ready for merging in!